database.go

```diff
@@ -140,6 +141,19 @@ func (db *Database) LockProtectedEntries() error {
 	return nil
 }
 
+func (db *Database) GetBinaries() *Binaries {
+	if db.Header.IsKdbx4() {
+		return &db.Content.InnerHeader.Binaries
+	}
+
+	return &db.Content.Meta.Binaries
+}
+
+// Find returns a reference to a binary in the database db with the same id as br, or nil if none is found
+func (db *Database) Find(br *BinaryReference) *Binary {
```

Comment:
`Find` as a name on the database level is a bit ambiguous - that could mean anything